packages/renderer/src/lib/preferences/PreferencesRegistriesEditing.svelte

```diff
@@ -45,11 +50,26 @@ const newRegistryRequest = $state<containerDesktopAPI.Registry>({
   secret: '',
 });
 
+// Preferred registries configuration property
+let preferredRegistriesProperty = $state<IConfigurationPropertyRecordedSchema | undefined>();
+let propertiesUnsubscribe: Unsubscriber;
+
 onMount(async () => {
   let providerSourceNames = await window.getImageRegistryProviderNames();
   if (providerSourceNames && providerSourceNames.length > 0) {
     defaultProviderSourceName = providerSourceNames[0];
   }
+
+  // Subscribe to configuration properties to get the preferred registries property
+  propertiesUnsubscribe = configurationProperties.subscribe(properties => {
```

Comment:
suggestion(non-blocking): I think you can also derive from this store $configurationProperties to build instead of subscribe preferredRegistriesProperty.